### DISCUSSION TITLE
Lista zamówień - tylko z fakturami

### DISCUSSION THREAD

**Discussion by akucaj** [allegro/allegro-api#6233] [Suggestion - verification]
### Jak otrzymać listę zamówień tylko z fakturami?

**Zasób:** 
api.allegro.pl/order/checkout-forms 

**Spodziewany filtr:** 
invoice.required=TRUE

**Wywołanie:**
`curl -X GET https://api.allegro.pl/order/checkout-forms?invoice.required=TRUE -H 'Authorization: Bearer ey...spmQ' -H 'accept: application/vnd.allegro.public.v1+json'`

Niestety wygląda na to że nie został zaimplementowany. Czy jest jakiś inny sposób?

**Comment by MartaNowaczyk** [comment]
Nie mamy takiego filtra. Twoje zgłoszenie przekazuję do specjalnej zakładki z sugestiami zmian w API Allegro.

**Comment by JamesFordV3** [comment]
podbijam sugestię - przydałaby się taka funkcjonalność
